subprojects/packagefiles/theora/examples/meson.build

```diff
@@ -0,0 +1,95 @@
+theora_libs = [libtheora]
+theoradec_libs = [libtheoradec]
+theoraenc_libs = [libtheoraenc, libtheoradec]
+
+compat_dep = dependency('compat', required: host_machine.system() == 'openbsd')
+
+if cc.has_header_symbol('unistd.h', 'getopt_long')
+  getopt_dep = disabler()
```

Comment:
A disabler acts like a viral NaN and causes all the later executable()s that try linking to it, to *also* become disabler objects and not be built. Is that the intention here?

Reply:
No, this should be a dummy `declare_dependency`.